.gitlab-ci.yml

```diff
@@ -24,39 +24,81 @@ variables:
     - tags
     - schedules
 
-
-.collect_artifacts:                &collect_artifacts 
+.collect_artifacts:                &collect_artifacts
   artifacts:
     name:                          "${CI_JOB_NAME}_${SCHEDULE_TAG:-${CI_COMMIT_REF_NAME}}"
     when:                          on_success
     expire_in:                     1 mos
     paths:
       - artifacts/
 
-test-linux:
+.cargo-check:                      &cargo-check
   stage:                           test
-  variables:
-    RUN_TESTS:                     all
-  script:
-    - scripts/gitlab/test-all.sh
+  before_script:
+    - sccache -s
+  after_script:
     - sccache -s
   tags:
     - linux-docker
 
-test-audit:
+
+cargo-check 0 3:
```

Comment:
nice trick :)

Reply:
yup, liked it, but without cache it's just a bit faster